config/initializers/datadog.rb

```diff
@@ -30,5 +30,36 @@
     # Enable ASM
     c.appsec.enabled = true
     c.appsec.instrument :rails
+
+    # Add dogstatsd_mapper_profiles field
+    c.dogstatsd_mapper_profiles = [
+      {
+        name: 'sidekiq',
+        prefix: 'sidekiq.',
```

Comment:
Should we prefix with vets-api instead of sidekiq?

Reply:
The value comes from the [datadog agent/sidekiq docs](https://docs.datadoghq.com/integrations/sidekiq/):

```
# dogstatsd_mapper_cache_size: 1000  # default to 1000
dogstatsd_mapper_profiles:
  - name: sidekiq
    prefix: "sidekiq."
    mappings:
      - match: 'sidekiq\.sidekiq\.(.*)'
        match_type: "regex"
        name: "sidekiq.$1"
      - match: 'sidekiq\.jobs\.(.*)\.perform'
        name: "sidekiq.jobs.perform"
        match_type: "regex"
        tags:
          worker: "$1"
      - match: 'sidekiq\.jobs\.(.*)\.(count|success|failure)'
        name: "sidekiq.jobs.worker.$2"
        match_type: "regex"
        tags:
          worker: "$1"
```

The prefix set to sidekiq works in the helm chart.

Reply:
As we create bulkheads, I imagine each app will get its own namespace. We should probably separate these now so metrics don't pollute future apps.

Reply:
Okay. What do you suggest we use as a prefix value then?

Reply:
We're using `vets_api` in other statsd metrics.

Reply:
I changed the prefix to `vets-api.`

Reply:
`vets_api`